### PR TITLE
Improve disable_vignettes

### PR DESCRIPTION
**Pull request by henriquegemignani**
With profiler overhead, this about 3.5 times faster.

### CODE REVIEW

File: src/open_samus_returns_rando/specific_patches/static_fixes.py [hunk 1]
```diff
@@ -362,3 +360,4 @@ def apply_static_fixes(editor: PatcherEditor) -> None:
     patch_a4_collision(editor)
     patch_a1_teleporter_crumbles(editor)
     disable_vignettes(editor)
+    raise SystemExit
```
Comment (Miepee) [inline src/open_samus_returns_rando/specific_patches/static_fixes.py:363]: nice debug

Comment (henriquegemignani) [reply src/open_samus_returns_rando/specific_patches/static_fixes.py:363]: oops

File: src/open_samus_returns_rando/specific_patches/static_fixes.py [hunk 2]
```diff
@@ -338,14 +338,12 @@ def disable_vignettes(editor: PatcherEditor) -> None:
...
                     # Check for the cc_name
                     if sg["str1"] == cc_name:
-                        for cc_group in range(len(sg["struct3"])):
-                            model_group = sg["struct3"][cc_group]["struct5"]
-                            for model in model_group:
-                                idx = model_group.index(model)
-                                for vignette in vignette_models:
+                        for cc_group in sg["struct3"]:
+                            model_group = cc_group["struct5"]
+                            for idx, model in reversed(list(enumerate(model_group))):
```
Comment (Miepee) [inline src/open_samus_returns_rando/specific_patches/static_fixes.py:343]: can't this be chaged to `... reversed(list(model_group))`?

Comment (henriquegemignani) [reply src/open_samus_returns_rando/specific_patches/static_fixes.py:343]: I want indices of the original list so `reversed` must be there. and you can't `reversed` and `enumerate`.

**Review by Miepee — APPROVED** [review]

**Pull request merged**